bonded-roles/src/main/java/bisq/bonded_roles/BondedRolesService.java

```diff
@@ -64,10 +64,13 @@ public static Config from(com.typesafe.config.Config config) {
     private final DifficultyAdjustmentService difficultyAdjustmentService;
     private final ReleaseNotificationsService releaseNotificationsService;
 
-    public BondedRolesService(Config config, PersistenceService persistenceService, NetworkService networkService) {
+    public BondedRolesService(Config config, PersistenceService persistenceService, NetworkService networkService,
+                              boolean skipMarketPriceService) {
         authorizedBondedRolesService = new AuthorizedBondedRolesService(networkService, config.isIgnoreSecurityManager());
         bondedRoleRegistrationService = new BondedRoleRegistrationService(networkService, authorizedBondedRolesService);
-        marketPriceService = new MarketPriceService(config.getMarketPrice(), persistenceService, networkService, authorizedBondedRolesService);
+        marketPriceService = skipMarketPriceService
```

Comment:
Better use Optional to make more explicit that field can be omitted.
If we use nullable fields we annotate with @Nullable (in domain code - in UI code that is relaxed as there is lot of usage of nullable).